Classes/Outlet/Pipe/EmailPipe.php

```diff
@@ -155,7 +187,11 @@ public function conduct($data)
      */
     protected function prepareEmail($data)
     {
-        $body = $this->getBody();
+        if ($this->getBodySection() !== null) {
+            $body = $this->view->renderStandaloneSection($this->getBodySection(), $data, true);
```

Comment:
Should we refactor this so that `$this->getBody()` result gets used if `$body` returned from View is empty?